xline-test-utils/src/lib.rs

```diff
@@ -197,6 +196,33 @@ impl Cluster {
         let decoding_key = DecodingKey::from_rsa_pem(public_key).unwrap();
         Some((encoding_key, decoding_key))
     }
+
+    pub fn default_config_with_quota_and_rocks_path(
+        path: PathBuf,
+        quota: u64,
+    ) -> XlineServerConfig {
+        let cluster = ClusterConfig::default();
+        let storage = StorageConfig::new(EngineConfig::RocksDB(path), quota);
+        let log = LogConfig::default();
+        let trace = TraceConfig::default();
+        let auth = AuthConfig::default();
+        let compact = CompactConfig::default();
+        XlineServerConfig::new(cluster, storage, log, trace, auth, compact)
+    }
+
+    pub fn default_rocks_config_with_path(path: PathBuf) -> XlineServerConfig {
+        Self::default_config_with_quota_and_rocks_path(path, default_quota())
+    }
+
+    pub fn default_rocks_config() -> XlineServerConfig {
+        let path = temp_dir().join(random_id());
+        Self::default_config_with_quota_and_rocks_path(path, default_quota())
+    }
+
+    pub fn default_quota_config(quota: u64) -> XlineServerConfig {
+        let path = temp_dir().join(random_id());
+        Self::default_config_with_quota_and_rocks_path(path, quota)
```

Comment:
I suggest to use `tempfile` for managing temp dirs.

Reply:
It doesn't make much sense to use `tempfile` here, because this temp directory will eventually be deleted on the `Cluster` drop, we don't need to rely on the auto-delete capability provided by the `tempfile` library